articles/code-management/about-code-management-csharp/about-code-management-csharp.md

```diff
@@ -0,0 +1,98 @@
+---
+uid: code-management.about-code-management-csharp
+---
+# About Code Management (C#)
+
+This article provides information about [Code Management](xref:code-management.about-code-management) done in a C# context using the `Intent.OutputManager.RoslynWeaver` Module.
+
+## Introduction
+
+In C# files it is quite common to notice this line of code near the top of the file:
+
+```csharp
+[assembly: DefaultIntentManaged(Mode.Fully)]
+```
+
+This attribute is located on a global scope which indicates that it will instruct Intent Architect to treat the whole file according to the configuration mode of `Fully` (in this case).
```

Comment:
Agreed. Will be reworded to say:

> This attribute is located on a global scope which indicates that it will instruct the Roslyn Weaver to by default treat the whole file according to the configuration mode of `Fully` (in this case). It may be overridden in certain parts of the file which will be addressed later.

Reply:
Still not ideal.

For:

> ...which indicates that it will instruct the...

Considering that one of the definitions of "indicate" is "suggest as a desirable or necessary course of action.", it's wrong to use the word here, instead just say:

> ...which instructs the...

For:

> ...treat the whole file according to the configuration mode of..

Because it's saying "whole file", it still seems like it's about treating the file holistically, which it's not. Essentially, it's instructing the RoslynWeaver that for each syntax node, unless otherwise specified, this is the IntentManaged instruction.